libs/template/application/oryx-app/oryx-app.component.spec.ts

```diff
@@ -8,7 +8,7 @@ import { siteProviders } from '@spryker-oryx/site';
 import { html } from 'lit';
 import { BehaviorSubject, Observable, Subject } from 'rxjs';
 import { SpyInstance } from 'vitest';
-import { rootAppComponent } from './root-app.def';
+import { oryxAppComponent } from './oryx-app.def';
 
 class MockRouterService implements Partial<RouterService> {
```

Comment:
For some reason can't comment this exact line:
`describe('RootAppComponent', ()`